Documentation/Testing/ExtensionTesting.rst

```diff
@@ -346,8 +332,8 @@ Let's run the tests:
 
     lolli@apoc /var/www/local/git/enetcache $ Build/Scripts/runTests.sh
     Creating network "local_default" with the default driver
-    PHP 7.2.11-3+ubuntu18.04.1+deb.sury.org+1 (cli) (built: Oct 25 2018 06:44:08) ( NTS )
-    PHPUnit 7.1.5 by Sebastian Bergmann and contributors.
+    PHP ....
```

Comment:
What about all these points?

Reply:
I wanted to avoid always having to replace the version numbers in the command output (because it the version is not really relevant there, but distracting if it is the wrong version). So I wanted to replace output with version numbers with something generic.

**I have exchanged it now for some actual output of using command in latest styleguide extension.**